addons/website_forum/static/src/js/website_forum.js

```diff
@@ -256,31 +255,24 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         if (!karma) {
             return;
         }
-
         ev.preventDefault();
+        if (session.is_website_user) {
+            this._displayAccessDeniedNotification(Markup(_t('<p>Oh no! Please <a href="/web/login">sign in</a> to vote</p>')));
```

Comment:
ah ok so you did it in this commit
shouldn't we do that in the commit in which we change the string ?

Reply:
ok, not a big deal :)